docs/troubleshooting-guide.md

```diff
@@ -70,7 +71,25 @@ To mitigate, please resize your VM in the resource group to a machine with at le
 ### Setup Cromwell on Azure for multiple users in the same Azure subscription
-This section is COMING SOON.
+Cromwell on Azure is designed to be flexible for single and multiple user scenarios. Here we have envisioned 4 general scenarios and demonstrated how they relate to your Azure account, Azure Batch service, Subscription ID, and Resource Groups, each depicted below.
+
+![Multiple Users FAQ](/docs/screenshots/multiple-users-configs.png)
+
+1) **The Individual User**: This is the current standard deployment configuration for Cromwell on Azure. No extra steps beyond the Quick Start guide are necessary.
+
+2) **The Lab**: This scenario is envisioned for small lab groups and teams sharing a common Azure resource (ie. a common bioinformatician(s) / data scientist(s) / compututational biologist(s) collaborating on projects from the same lab). Functionally, this setup does not differ from the "Individual User" configuration. We recommend a single "Cromwell Administrator" setup Cromwell on Azure for the group. (Be sure this user has the appropriate role(s) on the Subscription ID as outlined [here](https://github.com/microsoft/CromwellOnAzure#prerequisites). Once deployed, this "Cromwell Administrator" can grant access to the created Cromwell Storage Account via the [Azure Portal](https://ms.portal.azure.com/#home). This would allow granted users the ability to submit analysis jobs and retrieve results. It would also allow them the ability to view *any analysis* that has been run by the lab. As Cromwell submits all jobs to Azure Batch as one user, the billing for Cromwell on Azure usage would be collective for the entire lab, not broken down by individual users who submitted the jobs. 
+
+3) **The Research Group**: This scenario is envisioned for larger research groups where a common Azure subscription is shared, but users want/require their own instance of Cromwell on Azure. The initial Cromwell on Azure deployment is done as described in the [Quick Start](https://github.com/microsoft/CromwellOnAzure#Deploy-your-instance-of-Cromwell-on-Azure) guide. After the first deployment of Cromwell on Azure is done on the Subscription, subsequent users will need to specify a *separate Resource Group* **AND** *preexisting Azure Batch service name* that is currently being utilized by the pre-existing deployment(s) of Cromwell on Azure. See the invocation of the Linux deployment script for an example:  
+
+   (INSERT CODE LINES HERE)  
+
+   In this scenario, please note the lack of separation at the Azure Batch service level. While you will be able track resource usage independantly due to the separate Cromwell Users submitting analyses to Azure Batch (for your own tracking/internal billing purposes), anyone who has access to Azure Batch as a Contributor or Owner will be able to see ***everyone's*** Batch pools, and thus what they are running. For this scenario, we would recommend the Cromwell Administrator(s) be trusted personnel, such as your IT team.
```

Comment:
Should we elaborate on how to track various CoA instances that submit Batch jobs? Are Batch jobs tagged with the submitter id? cc @tonybendis